libsweep/include/Error.hpp

```diff
@@ -0,0 +1,12 @@
+#ifndef SWEEP_ERROR_2EADE195E243_H
+#define SWEEP_ERROR_2EADE195E243_H
+
+#include <stdexcept>
+
+namespace sweep {
+    struct ErrorBase : std::runtime_error {
```

Comment:
OK on the lower case, but removing `Base` will let it collide with the error defined in sweep.cc.